target/linux/ramips/dts/RT-AC51U.dts

```diff
@@ -0,0 +1,106 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "asus,rt-ac51u", "ralink,mt7620a-soc";
+	model = "Asus RT-AC51U";
+
+	gpio-leds {
+		compatible = "gpio-leds";
+				
+		power {
+			label = "rt-ac51u:blue:power";
+			gpios = <&gpio0 9 GPIO_ACTIVE_LOW>;			
+		};
+
+		activity {
+			label = "rt-ac51u:blue:activity";
```

Comment:
According to the manual the LED is the USB LED. Please name the LED to usb and setup the the usb trigger in ```target/linux/ramips/base-files/etc/board.d/01_leds```